Tasks/DownloadSecureFile/task.json

```diff
@@ -31,6 +31,14 @@
             "defaultValue": "",
             "required": true,
             "helpMarkDown": "Select the secure file to download to a temporary location on the agent. The file will be cleaned up after the build or release."
+        },
+        {
+            "name": "secureFilePath",
+            "type": "secureFile",
```

Comment:
The type should be "filePath" or may be "string". "secureFile" means it is a file stored on the server. The UI renders a different input control based on type, so this will show a drop down in the UI instead of a text box with a relative path from repo root.